final_prep/README.md

```diff
@@ -23,52 +23,57 @@ Some things to consider as you refactor include...
 
-- [ ] Standard Mod 1 class schedule (find the Mod 1 calendar [on the Back End curriculum site](https://backend.turing.edu/)). The detailed classes held on your actual start date may not yet be posted; you can look back at the previous inning to get a feel for what Weeks 1-3 will be like, but know it's not guaranteed to be exactly the same. Also, as you already know, class is held from 9-12 and 1-4MT, Monday-Friday.
-- [ ] Study/Project work time (usually about 25 hours/week outside of class time)
-- [ ] Personal, Health + Wellness
+- [x] Standard Mod 1 class schedule (find the Mod 1 calendar [on the Back End curriculum site](https://backend.turing.edu/)). The detailed classes held on your actual start date may not yet be posted; you can look back at the previous inning to get a feel for what Weeks 1-3 will be like, but know it's not guaranteed to be exactly the same. Also, as you already know, class is held from 9-12 and 1-4MT, Monday-Friday.
+- [x] Study/Project work time (usually about 25 hours/week outside of class time)
+> Study/Project time on these calendars exists in the white space and on weekends. I work best with benchmarks to have things completed by a certain time, rather than dedicated blocks. (Benchmarks will be added to calendar once I know what projects are/contain)
+- [x] Personal, Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
 - `Add Week 1 Screenshot Here`
+- ![4thweekmarchmod1](https://user-images.githubusercontent.com/99222755/156617578-c59b8a3b-efd8-45c9-b651-9821eaf83517.PNG)
+
 - `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- ![3rdweekMarchmod1](https://user-images.githubusercontent.com/99222755/156617713-55c50f94-b083-403f-9085-81dbbb656151.PNG)
 
+- `Add Week 3 Screenshot Here`
+- ![Aprilweek1mod1](https://user-images.githubusercontent.com/99222755/156617779-ae92e4ec-8081-40eb-9d65-a77c8d821216.PNG)
 ## Mentorship Prep
 
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 
-- [ ] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
-- [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+- [X] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
+- [x] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
+  - [x] Add link to your gist here: [DTR Gist Link](https://gist.github.com/SarahEHall/29d16752ef81e560f41ac1f82fb43106)
```

Comment:
Sai until now I didn't know that is what you go by. You do not need to use "Sarah" at Turing - we have to have legal names on the enrollment paperwork, but everywhere else it can be Sai (Slack, Zoom, etc)

Reply:
Ah, thank you so much! I wasn't sure, but that's a relief.  Also, I really appreciate your comments and encouragement, they always put a little extra pep in my step. :)